internal/database/database_test.go

```diff
@@ -424,3 +427,90 @@ func testGetAccounts(t *testing.T, db DB) {
 		assert.Equal(t, 0, len(accounts))
 	})
 }
+
+func testModifiedTimeUpdate(t *testing.T, db DB) {
```

Comment:
Can you add a docstring defining exactly what we are testing there or a better function name? (`testUpdateBookmarkUpdatesModifiedTime` ?)